pages/index.md

```diff
@@ -15,7 +15,7 @@ There are three core specifications currently.
 
 | Name | Version | Description |
 | --- | --- | --- |
-| [DSNP](DSNP/Overview.md) | 1.1.0 | The implementation-agnostic DSNP specification |
+| [DSNP](DSNP/Overview.md) | 1.2.0 | The implementation-agnostic DSNP specification |
```

Comment:
Does 1.2.0 supersede 1.1.0?

Reply:
Yes -- if anyone wants to know what 1.1.0 "was", they can go back to the relevant tag in git.

That said, if we remove or deprecate features, they should remain referenced in the spec with a note on the historical versions they were part of. We did this for Graph Change Announcement (separate PR recently merged).